retail-managment-system/rms-ui/src/app/pages/transaction/transaction.component.ts

```diff
@@ -28,6 +28,7 @@ export class TransactionComponent implements OnInit {
   currentData!: Transaction
   isButtonVisible: boolean = true
   submit: boolean = false;
+  index =0
```

Comment:
are you use this

Reply:
yes I am using that variable in displaying numbers in transaction.component.ts at the first column  "no."

Reply:
this index different for index in table